src/assets/scss/setup/_mixins.scss

```diff
@@ -0,0 +1,63 @@
+/*
+Use these breakpoint mixins (created around Angular Material breakpoints) for customizing your components/styling.
```

Comment:
These are handy, but I would not go into any of this more advanced SASS during the Angular presentation, just to prevent confusion. Don't even open up any of the SASS setup files--just wave your hands at them.

> The only proper way to make a martini is with ice-cold gin and a bow in the direction of France.

-Winston Churchill

A separate "This is how we SASS" workshop is a neat idea.